[2025-02-02T23:12:41.657+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T23:12:41.677+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-02-02T23:12:37.991166+00:00 [queued]>
[2025-02-02T23:12:41.687+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-02-02T23:12:37.991166+00:00 [queued]>
[2025-02-02T23:12:41.688+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-02T23:12:41.705+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_file_in_directory> on 2025-02-02 23:12:37.991166+00:00
[2025-02-02T23:12:41.712+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4551) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T23:12:41.714+0000] {standard_task_runner.py:72} INFO - Started process 4553 to run task
[2025-02-02T23:12:41.713+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_file_in_directory', 'manual__2025-02-02T23:12:37.991166+00:00', '--job-id', '382', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpvuld5bg5']
[2025-02-02T23:12:41.715+0000] {standard_task_runner.py:105} INFO - Job 382: Subtask check_file_in_directory
[2025-02-02T23:12:41.935+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-02-02T23:12:37.991166+00:00 [running]> on host 003b7fbea265
[2025-02-02T23:12:42.004+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_file_in_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T23:12:37.991166+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T23:12:37.991166+00:00'
[2025-02-02T23:12:42.005+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-02T23:12:42.005+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-02T23:12:42.006+0000] {logging_mixin.py:190} INFO - Current task name:check_file_in_directory state:running start_date:2025-02-02 23:12:41.678424+00:00
[2025-02-02T23:12:42.007+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-02T23:12:42.007+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T23:12:42.008+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-02T23:12:42.011+0000] {python.py:240} INFO - Done. Returned value was: verify_file_reliability
[2025-02-02T23:12:42.012+0000] {branch.py:38} INFO - Branch into verify_file_reliability
[2025-02-02T23:12:42.013+0000] {skipmixin.py:230} INFO - Following branch ('verify_file_reliability',)
[2025-02-02T23:12:42.024+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-02T23:12:42.050+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T23:12:42.051+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_file_in_directory, run_id=manual__2025-02-02T23:12:37.991166+00:00, execution_date=20250202T231237, start_date=20250202T231241, end_date=20250202T231242
[2025-02-02T23:12:42.060+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-02T23:12:42.060+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-02T23:12:42.061+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-02 23:12:37.998485+00:00
[2025-02-02T23:12:42.061+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-02T23:12:42.090+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T23:12:42.250+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-02T23:12:42.253+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
